### PR TITLE
Remove subprocessVersions

### PR DESCRIPTION
**Pull request by gskrobisz**
Removal of deprecated field, see `//TODO: field subprocessVersions is deprecate - to remove`

### CODE REVIEW

File: scenario-api/src/main/scala/pl/touk/nussknacker/engine/build/ScenarioBuilder.scala [hunk 1]
```diff
@@ -33,9 +33,6 @@ class ProcessMetaDataBuilder private[build](metaData: MetaData) {
   def slug(p: Option[String]) =
     new ProcessMetaDataBuilder(metaData.copy(typeSpecificData = RequestResponseMetaData(p)))
 
-  def subprocessVersions(subprocessVersions: Map[String, Long]) =
```
Comment (arkadius) [inline scenario-api/src/main/scala/pl/touk/nussknacker/engine/build/ScenarioBuilder.scala:36]: I think that we need a migration guide entry that this API method is unavailable anymore

File: common-api/src/main/scala/pl/touk/nussknacker/engine/api/MetaData.scala [hunk 2]
```diff
@@ -14,8 +14,7 @@ import scala.concurrent.duration.Duration
 // todo: MetaData should hold ProcessName as id
 @ConfiguredJsonCodec case class MetaData(id: String,
                                          typeSpecificData: TypeSpecificData,
-                                         additionalFields: Option[ProcessAdditionalFields] = None,
-                                         subprocessVersions: Map[String, Long] = Map.empty) { //TODO: field subprocessVersions is deprecate - to remove
```
Comment (arkadius) [inline common-api/src/main/scala/pl/touk/nussknacker/engine/api/MetaData.scala:18]: This is an api module so migration guide entry is necessary

**Review by arkadius — COMMENTED** [review]
1. Have you tested, that scenario jsons witch obsolete `subprocessVersions` field will be accepted by import mechanism or we check there if json doesn't contain redundant fields?
2. Similar question regarding database. migration is not need?

File: scenario-api/src/main/scala/pl/touk/nussknacker/engine/build/ScenarioBuilder.scala (hunk 1 above)
Comment (gskrobisz) [reply scenario-api/src/main/scala/pl/touk/nussknacker/engine/build/ScenarioBuilder.scala:36]: Fixed.

File: common-api/src/main/scala/pl/touk/nussknacker/engine/api/MetaData.scala (hunk 2 above)
Comment (gskrobisz) [reply common-api/src/main/scala/pl/touk/nussknacker/engine/api/MetaData.scala:18]: Fixed.

**Comment by gskrobisz** [comment]
> 1. Have you tested, that scenario jsons witch obsolete `subprocessVersions` field will be accepted by import mechanism or we check there if json doesn't contain redundant fields?
Yes, json deserailzation ignores unknown fields (confirmed in test).

>     2. Similar question regarding database. migration is not need?
Database migration is not required. Obsolete fields are ignored when json is retrieved from repository.

**Review by arkadius — APPROVED** [review]
LGTM but let's product team add one more +1

**Pull request merged**